### DISCUSSION TITLE
Custom advanced module - inventory variables

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15287] [How to] [answered]
PDF Template by default has set inventory/products custom variables. Emails Templates and Workflow emails not.
How to use Inventory line items in above for custom advanced module?

**Comment by garryatbrighton** [comment]
Having the same problem. How to show inventory from custom module in Email templates? Experiments with texparsers not working somehow.

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
what exactly do you want to show in the email from the inventory

**Reply by s2b-git** [reply]
It works if carefully duplicate functions in app/TextParsers

**Comment by garryatbrighton** [comment]
Custom module (advanced) have line items (services) and this needs to be shown in the way ProductsTableNew class does.
Throwing ProductsTableNew.php to texparsers in custommodule folder plus changing filename, class name etc still doent show line items.
Guessing there is some little thing I cannot get
